tools/bigquery-query-plan-exporter/setup.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import io
+import os
+import setuptools
+
+
+name = 'bqutil'
+description = 'Utility to export BigQuery Jobs and Query Plans'
+version = '0.1.0'
+release_status = 'Development Status :: 3 - Alpha'
+dependencies = [
+    'google-cloud-bigquery >= 1.9.0'
+]
+extras = {}
```

Comment:
Is there a reason for defining this? I think we can drop this and remove `extras_require` kwarg from setup call below.